src/extensions/default/InlineColorEditor/CSSProperties.json

```diff
@@ -0,0 +1,230 @@
+{
+    "align-content":               {"values": ["center", "flex-end", "flex-start", "space-around", "space-between", "stretch"]},
```

Comment:
On lines 50, 51 of `src/extensions/default/InlineColorEditor/main.js` you basically have to do 2 `if` checks to narrow things down, so yes, I'd just preemptively do this and only include the properties you know you care about for color in the file.  Yes, renaming the file is probably good.